clusterloader2/testing/node-throughput/config.yaml

```diff
@@ -33,7 +34,7 @@ steps:
       apiVersion: v1
       kind: ReplicationController
       labelSelector: group = latency
-      operationTimeout: 15m
+      operationTimeout: {{$OPTS_TIMEOUT}}
```

Comment:
Maybe name the variable "REPLICATION_TIMEOUT" or "OPERATION_TIMEOUT" ?

OPTS is not clear / specific enough.